nengo/simulator.py

```diff
@@ -151,15 +151,20 @@ def __init__(
         else:
             self.model = model
 
-        if network is not None:
-            # Build the network into the model
-            self.model.build(network, progress_bar=self.progress_bar)
+        with ProgressTracker(
```

Comment:
This will break the progressbar for Nengo OCL. Just pointing it out to remind myself to fix it once this is merged.